readme.md

```diff
@@ -54,7 +59,7 @@ console.log(results);
 
 Your Answer:
 ```text
-
+Bob has quotations marks around it. In order to show an author model on the second line you'd have to make "Bob" a function.
 ```
```

Comment:
The issue here is that `.find` is a synchronous function. As such, `results` will be undefined when we try to print it to the console. We can solve this by passing in a callback function to `.find` like so...

```js
var results = AuthorModel.find({name: "Bob"}, function(results){
  console.log(results);
});
```